internal/crdschema/crd.go

```diff
@@ -232,7 +231,7 @@ func filterNonBreaking(diffMap map[string]*diff.Diff) map[string]*diff.Diff {
 	return diffMap
 }
 
-func ignoreOptionalNewProperties(sd *diff.SchemaDiff) {
+func ignoreOptionalNewProperties(sd *diff.SchemaDiff) { // nolint:gocyclo
```

Comment:
I highly recommend enabling `nolintlint`. It requires folks to explain why they're adding a `nolint` directive, and also catches `nolint` directives that are ineffective, e.g. because a function that used to be over the cyclomatic complexity level later dropped below it.

https://github.com/crossplane/crossplane/commit/03609c4d0eae53205269b7806f16a6fc4389b233

Reply:
Done. Also broken some high cyclomatic complexity functions.